components/Carbon.js

```diff
@@ -112,7 +113,38 @@ class Carbon extends React.Component {
             value={this.props.children}
             options={options}
           />
+          <div id="container-bg">
```

Comment:
https://www.dropbox.com/s/68ufy8w1d5gtmrz/Screenshot%202017-10-04%2022.38.43.png?dl=0

This pixel is still a little off homie

Reply:
Agh, okay. Will fix tonight. Good catch.